lib/google_drive/spreadsheet.rb

```diff
@@ -131,5 +131,18 @@ def batch_update(requests)
         @session.sheets_service.batch_update_spreadsheet(id, batch_request)
       batch_response.replies
     end
+
+    # Append values to a spreadsheet by first searching for a data table at a range,
+    # then appending the specified values at the end of this data table.
+    def append(range_name, values, override_params = {})
```

Comment:
Can you rename this to `append_values`? `append` sounds ambiguous and can mean "append a worksheet".

Reply:
Agreed, thanks.